packages/happo-core/src/getLastResultSummary.js

```diff
@@ -3,10 +3,12 @@ const path = require('path');
 
 const { config } = require('./config');
 
-module.exports = function getLastResultSummary() {
+function getLastResultSummary() {
   const resultSummaryJSON = fs.readFileSync(
     path.join(config.snapshotsFolder, config.resultSummaryFilename),
     'utf8',
   );
   return JSON.parse(resultSummaryJSON);
-};
+}
+
+module.exports = getLastResultSummary;
```

Comment:
that's fine. i wasn't sure since i saw a mixture of `require` / `module.exports` and `import` / `export`